File: src/src/Account.java
public class Account {
    private long account_number;
    private double amount = 0;
    private char account_type;
    private double amount_max;

    public Account(long account_numbe, char account_types) {
        if (account_types!='A' && account_types!='B' && account_types!='C') {
            System.out.println("\n TIPO NO VALIDO, CUENTA VACIA");
            this.account_number=0;
            account_type=' ';
        } else{
            this.account_number = account_numbe;
            account_type = account_types;
            if (account_type == 'A') {
                amount_max = 50000;
            } else if (account_type == 'B') {
                amount_max = 100000;
            }
        }

    }




    public long getAccount_number() {
        return account_number;
    }

    public void setAccount_number(long account_number) {
        this.account_number = account_number;
    }

    public double getAmount() {
        return amount;
    }

    public char getAccount_type(){
        return account_type;
    }

    public void setAmount(double amounts) {
        if (account_type=='A' || account_type=='B') {
            if (amount < amount_max && amount + amounts < amount_max) {
                amount += amounts;
            }  else{
                System.out.println("DEMASIADO VALOR");
            }


        }else if (account_type=='C'){
            amount += amounts;
        }
    }

    public void extamount ( double ext){
        if (account_type=='A') {
            if (ext>=1000 && ext<amount_max && ext<amount){
                amount-=ext;
            } else{
                System.out.println("CANTIDAD INADECUADA");
            }
        } else if (account_type=='B'){
            if (ext>=5000 && ext<amount_max && ext<amount){
                amount-=ext;
            } else{
                System.out.println("CANTIDAD INADECUADA");
            }
        } else if (account_type=='C') {
            if (ext>=10000 && ext<amount_max && ext<amount){
                amount-=ext;
            } else{
                System.out.println("CANTIDAD INADECUADA");
            }
        }
    }
}

File: Account.py

class Account:
    account_number=0
    amount = 0
    account_type= ''
    ammount_max=0

    def __init__(self,accoun_num,type):
        if type!='A' and type!='B' and type!='C':
            print("TIPO NO VALIDO, CUENTA NO CREADA ")
        else:
            self.account_number=accoun_num
            self.account_type=type
            if type=='A':
                self.ammount_max = 50000
            elif type=='B':
                self.ammount_max = 100000


    def get_acc_number(self):
        return self.account_number

    def get_amount(self):
        return self.amount

    def get_acount_type(self):
        return self.account_type

    def setAmount (self,quantity):
        if self.account_type=='A' or self.account_type=='B':
            if self.amount<self.ammount_max and self.amount + quantity <self.ammount_max:
                self.amount += quantity
        elif self.account_type=='C':
                self.amount += quantity
        else:
                print("VALOR NO ADECUADO")


    def extamount(self, quant):
        if self.account_type=='A':
            if quant>=1000 and quant<self.ammount_max and quant<self.amount:
                self.amount-=quant
            else:
                print("CANTIDAD INADECUADA")
        elif self.account_type=='B':
            if quant >= 5000 and quant < self.ammount_max and quant < self.amount:
                self.amount -= quant
            else:
                print("CANTIDAD INADECUADA")
        elif self.account_type=='C':
            if quant >= 10000 and quant < self.amount:
                self.amount -= quant
            else:
                print("CANTIDAD INADECUADA")
